### PR TITLE
chore: provision

### PR DESCRIPTION
**Pull request by gcedo**

### CODE REVIEW

**Comment by umbobabo** [comment]
We shouldn't avoid to create a major here. It will be painful update all the component manually.
2 major of this component cannot coexist on the same project.

**Comment by keithamus** [comment]
it's not really a breaking change for us: we import using `@economist/typography` which will still work given that the package.json points to `lib/index.css` as its [`style`](https://github.com/economist-components/component-typography/pull/26/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R16). AFAICT there are no "breaking" changes here.

**Comment by keithamus** [comment]
Rebase please @gcedo :smile:

**Comment by gcedo** [comment]
@keithamus rebased :blush:

File: src/example.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable */
```
Comment (keithamus) [inline src/example.css]: Hmm this doesn't look very good. We should just fix lint errors, not disable the linter.

File: src/font-face.css [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable */
```
Comment (keithamus) [inline src/font-face.css]: Same here. Please fix the lint errors instead of disabling the linter.

File: src/index.css [hunk 3]
```diff
@@ -1,4 +1,5 @@
-@import "font-face.css";
+/* stylelint-disable */
```
Comment (keithamus) [inline src/index.css]: www.takescoop.com

Comment (keithamus) [inline src/index.css]: Same here. Please fix the lint errors instead of disabling the linter.

**Comment by gcedo** [comment]
@keithamus fixed

**Comment by keithamus** [comment]
Good enough :smile:

**Pull request merged**